packages/haste-task-copy/test/index.spec.js

```diff
@@ -0,0 +1,102 @@
+const fs = require('fs');
+const path = require('path');
+const tempy = require('tempy');
+const copy = require('../src');
```

Comment:
this should be after 3rd party deps (mkdirp)

Reply:
Agree, will be fixed